src/src/WebServer/DevicesPage.cpp

```diff
@@ -540,9 +540,11 @@ void handle_devicess_ShowAllTasksTable(uint8_t page)
 
       if (validDeviceIndex(DeviceIndex)) {
         if (Settings.TaskDeviceDataFeed[x] != 0) {
+          #ifdef FEATURE_ESP_P2P
           // Show originating node number
           const uint8_t remoteUnit = Settings.TaskDeviceDataFeed[x];
           format_originating_node(remoteUnit);
+          #endif
```

Comment:
Not sure if you should hide that, if the task was originally created in an older config that was still using P2P? But maybe it should be shown differently.

Reply:
the question then how differently?

Reply:
Maybe not different, if it will properly show the source without any dependency, then that's fine, if it tries to get the IP or name of the node, then that should probably be avoided.